jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/DefaultCredentialRetrievers.java

```diff
@@ -112,10 +114,15 @@ public DefaultCredentialRetrievers setCredentialHelperSuffix(
     if (knownCredentialRetriever != null) {
       credentialRetrievers.add(knownCredentialRetriever);
     }
-    if (credentialHelperSuffix != null) {
-      credentialRetrievers.add(
-          credentialRetrieverFactory.dockerCredentialHelper(
-              DockerCredentialHelper.CREDENTIAL_HELPER_PREFIX + credentialHelperSuffix));
+    if (credentialHelper != null) {
+      if (Files.exists(Paths.get(credentialHelper))) {
```

Comment:
I think we might want to be careful about a credential helper suffix matching a file in the cwd. For example, if there's a file called `gcr` in the directory, this would try to use that as the credential helper executable. I think we should limit this to only check as a path if `credentialHelper` contains at least one file separator.